_includes/homepage/showcase.html

```diff
@@ -1,45 +1,41 @@
-<section class="bp-section" style="background-color:#6031b6" id="showcase">
+<section class="bp-section {{ gray_container_class }}">
     <div class="bp-container padding--bottom--lg">
         <div class="row">
             <div class="col is-8 is-offset-2 has-text-centered padding--top--xl">
-                <p class="padding--bottom eyebrow is-uppercase has-text-white">
-                    SHOWCASE
+                <p class="padding--bottom eyebrow is-uppercase">
+                    {{ section.showcase.subtitle }}
                 </p>
-                <h1 class="padding--bottom has-text-white"><b>Websites generated using Isomer</b></h1>
-                <p class="has-text-white">Our websites are compliant-by-design, so that you can focus on your content</p>
+                <h1 class="has-text-secondary padding--bottom"><b>{{ section.showcase.title }}</b></h1>
+                <p>{{ section.showcase.description }}</p>
             </div>
         </div>
-        <div class="row has-text-centered margin--top padding--bottom">
-            <div class="col is-one-third">
-                <a href="https://www.hlb.gov.sg" target="_blank" class="is-media-card">
-                    <div class="image-hoverable">
-                        <img class="isomer-example" src="{{ site.baseurl }}/images/hlb-example.png" />
-                        <div class="image-hover-overlay"></div>
-                        <div class="bp-button is-white is-outlined is-uppercase image-hover-button search-button">Visit Site</div>
+        <div class="row has-text-centered margin--top padding--bottom showcase showcase-{{ section.showcase.colcount }}">
+            {%- for entry in section.showcase.entries -%}
+            <div class="col">
+                {%- if entry.url -%}
+                {%- assign url_input = entry.url -%}
+                {%- include functions/external_url.html -%}
+                <a {{anchor}} class="is-media-card" target="_blank">
+                {%- endif -%}
+                <div class="image-hoverable">
+                    <img src="{{ site.baseurl }}{{ entry.image }}" />
+                    <div class="image-hover-overlay">
+                        <div class="image-hover-overlay-container">
+                            {%- if entry.description -%}
+                            <div>{{ entry.description }}</div>
+                            {%- endif -%}
+                            {%- if entry.button -%}
+                            <div class="bp-button is-white is-outlined is-uppercase search-button">{{ entry.button }}</div>
+                            {%- endif -%}
+                        </div>
```

Comment:
Did we have a wireframe for the showcase design? The behavior on-hover leaves a lot to be desired. To me it's jarring that a colored background and text is overlaid on the picture. IMO It would be more intuitive for the bottom Entry Title to be the button instead

Reply:
I took the cue from the showcase on isomer.gov.sg. I largely agree with your point for the button, though I still think hover text would be useful (look at the subtext clutter in the healthcare corps website for comparison)

Reply:
I think it works well for isomer.gov.sg because the color of the overlay was the same as the background. So instead of using `$primary` can we set it to be the same as the surrounding background?

Reply:
Yea, that works. I'll make the relevant changes

Reply:
![image](https://user-images.githubusercontent.com/10572368/82166187-525fca00-98ea-11ea-8435-b6c47be42d39.png)

Reply:
Sorry I forgot to mention this before, can we also remove the “description” field?

Reply:
Hmm. Gonna push back on that. Some people may want a description to go along with the title

Reply:
Can you give an example of a description that would be apt in this use case?

@lisatjide @prestonlimlianjie thoughts? I don’t feel too strongly about this, just think that it’s unnecessary information

Reply:
See the Healthcare Corps page, which supplements the title with a bit more context

Reply:
I agree with the removing of description part that jiehao pointed. also after a second look, having two rows of items may not be the best idea as well. 

Using a generic black filter on hovering may look better cuz it looks a bit weird if it is blending in with the white background although purple looks fine on isomer.gov.sg

We should get a designer to take a look at this before we push this to prod.

Reply:
Aren't the number of rows dependent on the number of columns specified by the user? Surely this is beyond the user's control?